config.json

```diff
@@ -783,6 +783,17 @@
       ],
       "unlocked_by": null,
       "uuid": "ce270a34-add1-422c-bb86-53b310f05e27"
+    },
+    {
+      "core": false,
+      "difficulty": 5,
```

Comment:
This doesn't quite seem like a 5 to me. There aren't really any weird edge cases going on. Once you figure out the basic algorithm, then you're good to go. Maybe a 4?